test/tests/outputs/png/wedge.i

```diff
@@ -17,6 +17,20 @@
   [../]
 []
 
+[AuxVariables]
+  [tew_u]
+  []
+[]
+
+[AuxKernels]
+  [tew_u]
```

Comment:
Now that's funny. I'm not sure if you are just playing with the homophone here or not. I suggested "two_u" because this variable is double the solution of u (2*u) or two_u, "tew_u" wasn't quite what I was after - haha

Reply:
Yeah, I thought it would be fun to have that homophone in there for one commit :). I'll switch it out now.